packages/elements/src/interactive-chart/index.ts

```diff
@@ -734,6 +718,29 @@ export class InteractiveChart extends ResponsiveElement {
     this.createRowLegend(this.rowLegend, param);
   };
 
+  /**
+   * Callback uses for sizeChange event
+   * @returns {void}
+   */
+  private onTimeScaleSizeChange = (): void => {
+    this.handleLegendLeftPosition();
+  };
+
+  /**
+   * Handle left position of legend
+   * @returns {void}
+   */
+  private handleLegendLeftPosition (): void {
+    const position = this.getPriceScalePosition();
+    if (position === 'left' || position === 'two-price') {
+      const leftPriceScaleWidth = this.chart?.priceScale('left').width() || 0;
+      this.legendContainer.style.left = `${leftPriceScaleWidth + this.DEFAULT_LEGEND_LEFT_POSITION}px`;
+    }
+    else {
+      this.legendContainer.style.left = '15px';
```

Comment:
Why don't use `this.DEFAULT_LEGEND_LEFT_POSITION` instead of `15`.
and should it possible to fix to use 
```javascript
this.legendContainer.style.left = `${leftPriceScaleWidth + this.DEFAULT_LEGEND_LEFT_POSITION}px`;
```
for any case instead of use if else condition?

Reply:
Agree. I will update it.

Reply:
Updated.